### DISCUSSION TITLE
Realtime CHANNEL_ERROR when subscribing - javascript

### DISCUSSION THREAD

**Discussion by kevinco26** [supabase/supabase#19263] [Questions]
I have couple projects in my organization. I'm testing realtime events and one of my projects works perfectly. I'm able to connect using the sdk: 
const supabase = createClient(supaUrl, supaAnonKey)
  supabase
    .channel('test')
    .subscribe((status)=>{console.log(status)})

however, I was testing on a new project I have in my organization and I'm using the exact same javascript code but substituting the values for the supabase url and supabase anonkey and several things happen
1. I get CHANNEL_ERROR
2. It infinitely tries to subscribe producing CHANNEL_ERROR always
3. It outputs CHANNEL_ERROR twice before retrying within the next second (I'm not sure why the status/subscription is happening twice)
Also, I don't see documentation for how to access the error, would be good to have the error message but when I try  
.subscribe((status,err)=>{console.log(err)}) it says undefined .... (even though that's what the repo has?)

Again, when I use the config of my other project, it works perfectly. Subscribes once and that's it.

**Comment by GaryAustin1** [comment]
There is https://realtime.supabase.co/inspector/new you can run to see if the instance is running OK or your code is the difference.

**Comment by BellCubeDev** [comment]
For anyone experiencing this in the future, make sure your API key is correct. I must've shaved off a character when I copied mine the first time and it caused this problem.
